airflow/dag_processing/processor.py

```diff
@@ -168,18 +168,8 @@ def _handle_dag_file_processing():
             result_channel.send(result)
 
         try:
-            DAG_PROCESSOR_LOG_TARGET = conf.get_mandatory_value("logging", "DAG_PROCESSOR_LOG_TARGET")
```

Comment:
I believe the logic here is since scheduler dump logs to both stdout and file without LOG_TARGET variable then why dag processor needs one. 

So I wonder why the dag processor log logic was designed this way in the first place?
(Also not sure if we can just ignore `DAG_PROCESSOR_LOG_TARGET` that way.. we may need to set newsfragment)